ShipEngineSDK/Api/AccountApi.cs

```diff
@@ -192,9 +192,10 @@ public async Task<GetAccountSettingsImagesResponseBody> CreateAccountImage(HttpC
 
         requestOptions.Operation = "AccountApi.CreateAccountImage";
 
-        var result = await SendHttpRequestAsync<GetAccountSettingsImagesResponseBody>(HttpMethods.Post, requestOptions, methodClient, _config, cancellationToken);
-
-        return result;
+        var (data, response) = await GetHttpResponse<GetAccountSettingsImagesResponseBody>(HttpMethods.Post, requestOptions.FullPath(), requestOptions.Data, methodClient, _config, cancellationToken);
+        var headers = response.Headers.ToDictionary(h => h.Key, h => h.Value.FirstOrDefault(),
+            StringComparer.InvariantCultureIgnoreCase);
+        return new ShipEngineResponse<GetAccountSettingsImagesResponseBody>(data, response.StatusCode, headers);
```

Comment:
Am I reading this wrong or does this make the data inaccessible?  From what I saw, the `ShipEngineResponse` holds the data in a private field.